include/boost/geometry/extensions/gis/projections/impl/pj_init.hpp

```diff
@@ -204,8 +205,14 @@ inline parameters pj_init(R const& arguments, bool use_defaults = true)
 
     if (! s.empty())
     {
-        // TODO: IMPLEMENT SPLIT
-        pin.to_meter = atof(s.c_str());
+        std::size_t const divisor = s.find('/');
+        if (divisor == std::string::npos)
+            pin.to_meter = std::atof(s.c_str());
+        else
+        {
+            std::string const numerator(s.substr(0, divisor)), denominator(s.substr(divisor + 1));
+            pin.to_meter = std::atof(numerator.c_str()) / std::atof(denominator.c_str());
```

Comment:
It can come from pj_units, but can also come from a line as
```bg::projections::parameters par = bg::projections::detail::pj_init_plus(parameters);```
where a user enters ``` to_meter=1/10```